.github/workflows/test.yml

```diff
@@ -260,6 +260,13 @@ jobs:
           os: macos-13
           arch: x86_64
           action: test
+        - name: 'WAMR on Windows/x86_64'
+          engine: 'wamr'
```

Comment:
Erm... We have `wamr-interp` and `wamr-jit`, but there is no `engine=wamr`, so this shouldn't even work.

Reply:
That's....interesting. Maybe it uses nullvm when the engine is invalid? Should probably error there. Will do some more testing later

Reply:
It builds, because bazel configs are pattern matching, and `engine=wamr` is simply a no-op.

Could you try with `wamr-interp`?

Reply:
Yep; added one for wamr-jit to see if either works